R/httr2.R

```diff
@@ -52,14 +52,17 @@ chat_perform_async_value <- function(provider, req) {
   promises::then(req_perform_promise(req), resp_body_json)
 }
 
-on_load(chat_perform_async_stream <- coro::async_generator(function(provider, req, polling_interval_secs = 0.1) {
```

Comment:
Yessss, get out of here, `polling_interval_secs`!

Reply:
🐎